

class BaseException(RuntimeError):
    pass

class GeneralError(BaseException):
    pass